test.js

```diff
@@ -247,9 +251,7 @@ test('is.primitive', t => {
 		Symbol('🦄')
 	];
 
-	for (const el of primitives) {
-		t.true(m.primitive(el));
-	}
+	primitives.forEach(el => t.true(m.primitive(el)));
```

Comment:
I prefer how it was here. It was more readable and more efficient.

Reply:
Accepted, will revert this